webapp/channels/src/i18n/en.json

```diff
@@ -1390,11 +1390,11 @@
   "admin.license.remove": "Remove",
   "admin.license.removing": "Removing License...",
   "admin.license.renewalCard.description.contact_sales": "Renew your {licenseSku} license by contacting sales to avoid any disruption.",
-  "admin.license.renewalCard.licensedUsersNum": "**Licensed Users:** {licensedUsersNum}",
   "admin.license.renewalCard.licenseExpired": "License expired on {date, date, long}.",
   "admin.license.renewalCard.licenseExpiring": "License expires in {days} days on {date, date, long}.",
   "admin.license.renewalCard.reviewNumbers": "Review your numbers below to ensure you renew for the right number of users.",
-  "admin.license.renewalCard.usersNumbers": "**Active Users:** {activeUsersNum}",
+  "admin.license.renewalCard.usersNumbers_active": "<strong>Active Users:</strong> {activeUsersNum}",
+  "admin.license.renewalCard.usersNumbers_licensed": "<strong>Licensed Users:</strong> {licensedUsersNum}",
```

Comment:
please update with the latest translation here, its different from the newly added one